docs/documentation/server_admin/topics/realms/cross-domain-trust.adoc

```diff
@@ -0,0 +1,21 @@
+[[cross_domain_trust]]
+=== Configuring Cross-Domain Trusts
+
+Cross-domain trusts can be defined at the realm level to specify trusted security domains external to Keycloak. This set of trusted
+domains is used by other features, such as the <<_assertion_grant,JWTs as Authorization Grants>> feature which allows external entities
+to request access tokens for users.
+
+To configure a trusted domain:
```

Comment:
I'd imagine there would be a need to be able to lock down things further to have checks on claims within the assertions. For example if a different Keycloak server is the trusted domain, then since Keycloak uses JWTs for refresh, ID, and access tokens, there would need to be some mechanism to for example only permit ID tokens. For other providers there could be other claims that would need to be checked.